BrightID/src/components/Tasks/TaskCard.tsx

```diff
@@ -21,12 +22,17 @@ type TaskCardProps = {
 };
 
 function TaskCard(props: TaskCardProps) {
-  const { title, description, fulfilled, url, onClick } = props;
+  const { title, description, fulfilled, url, onClick, id } = props;
+  const navigation = useNavigation();
 
   const desc = url ? (
     <TouchableOpacity
       onPress={() => {
-        Linking.openURL(url);
+        if (id === 'bitu_verification') {
+          navigation.navigate('BituVerification');
+        } else {
+          Linking.openURL(url);
+        }
```

Comment:
We should make this more generic, so any task can either be a url or trigger navigation:
- Add optional property for a navigation target (`navigationTarget?: string`) to TaskCardProps
- When rendering the `desc` element:
  - is there a navigation target? -> Then do `navigate(navigationTarget) on press
  - else is there a url? -> Then do `Linking.openUrl()` on press
  - else -> plain text, not pressable